src/games/connections.js

```diff
@@ -0,0 +1,37 @@
+import Game from './game.js'
+
+const messageParser = (message) => {
+  const match = message.content.match(/Connections\s\sPuzzle\s\#(\d+)\s([\uD83D\uDFE6-\uDFEA\s]+)/)
```

Comment:
sku ønske det fantest en eller annen regex builder i js eller noe?

Reply:
syns d veldig leselig d der eg :)

Games forventer nå bare en `messageParser(message) -> { day, score }` funksjon for message validation, så du trenger ikke å bruke regex i fremtiden

Reply:
så _kan_ bruke regex men mulig å gjøre det uten?

Reply:
yep, eg bare liker regex fordi d tar lite plass og må isje fakke med for mye sånn trimming og array indices og split osv